### PR TITLE
boards: stm32h747i_disco: Use default tick rate

### LINKED ISSUE #18228: stm32h747i_disco: Fix SYS_CLOCK_TICKS_PER_SEC
**Describe the bug**
stm32h747i_disco was under review when https://github.com/zephyrproject-rtos/zephyr/pull/16782 got merged.
As a consequence it is using old default tick rate. This should be fixed.

### PR DESCRIPTION
**Pull request by erwango**
Remove SYS_CLOCK_TICKS_PER_SEC configuration to use default value.

Fixes #18228

Signed-off-by: Erwan Gouriou <erwan.gouriou@linaro.org>

### CODE REVIEW

**Review by carlescufi — APPROVED** [review]

**Review by galak — APPROVED** [review]

**Review by ioannisg — CHANGES_REQUESTED** [review]
I read the commit text body and I can't see how this patch removes the SYS_CLOCK_TICKS_PER_SEC; it seems to me it sets it to 100.

File: boards/arm/stm32h747i_disco/stm32h747i_disco_m4_defconfig [hunk 1]
```diff
@@ -5,7 +5,7 @@ CONFIG_BOARD_STM32H747I_DISCO_M4=y
 CONFIG_SOC_SERIES_STM32H7X=y
 CONFIG_SOC_STM32H747XX=y
 CONFIG_CORTEX_M_SYSTICK=y
-CONFIG_SYS_CLOCK_TICKS_PER_SEC=1000
+CONFIG_SYS_CLOCK_TICKS_PER_SEC=100
```
Comment (ioannisg) [inline boards/arm/stm32h747i_disco/stm32h747i_disco_m4_defconfig]: IMHO, "default" means, you wouldn't have to set it here :)

Comment (erwango) [reply boards/arm/stm32h747i_disco/stm32h747i_disco_m4_defconfig]: Doh! Git amend issue (Besides 100 is wrong), update on its way.

Comment (erwango) [reply boards/arm/stm32h747i_disco/stm32h747i_disco_m4_defconfig]: Thanks @ioannisg !

**Review by ioannisg — APPROVED** [review]

**Pull request merged**